presto-main/src/main/java/io/prestosql/sql/planner/LocalDynamicFiltersCollector.java

```diff
@@ -26,20 +32,26 @@
      * (e.g. in case of co-located joins).
      */
     @GuardedBy("this")
-    private TupleDomain<Symbol> predicate;
-
-    public LocalDynamicFiltersCollector()
-    {
-        this.predicate = TupleDomain.all();
-    }
+    private Map<Symbol, Domain> dynamicFilterDomainsResult = new HashMap<>();
 
-    public synchronized TupleDomain<Symbol> getPredicate()
+    public synchronized TupleDomain<Symbol> getPredicate(Collection<Symbol> probeSymbols)
```

Comment:
Make `probeSymbols` a `Set` to explicitly avoid duplicates

Reply:
done